### DISCUSSION TITLE
Code Coverage for Unit Test in Angular

### DISCUSSION THREAD

**Discussion by hebaibrahim76** [cypress-io/cypress#24336] [Questions and Help]
Hi, I'm trying to get a coverage report for unit testing in angular
For the instrumentation I'm using ngx-build-plus

Here's my repo https://github.com/hebaibrahim76/Unit-Testing

I added an add.ts file that has a simple addition function
I'm testing that function in cypress/e2e/add.cy.ts

The test is passing, but the coverage is being 0%

**Comment by geromegrignon** [comment]
Hi @hebaibrahim76, I never used coverage but based on your coverage config, it sounds like it excludes **cy.ts** files:

```
exclude: [
            /\.(e2e|spec|cy)\.ts$/,
            /node_modules/,
            /(ngfactory|ngstyle)\.js/
          ]
```

**Reply by hebaibrahim76** [reply]
@geromegrignon , I've already tried not adding it to the exclusions before, but I still faced the same issue.
I removed it now, and coverage is still 0%
